libs/wallet-ui/src/components/collapsible-panel/collapsible-pabel.spec.tsx

```diff
@@ -0,0 +1,45 @@
+import { fireEvent, render, screen } from '@testing-library/react'
```

Comment:
**chore (non-blocking)**: This should be `collapsible-panel.spec.tx` not `-pabel-`

Reply:
I've got an immediate follow up to this PR I am going to address this in there